glide.yaml

```diff
@@ -17,6 +17,8 @@ import:
   subpackages:
   - gomock
 - package: github.com/uber/tchannel-go
-  version: ^1.0.9
+  version: ^1.1.0
+- package: github.com/opentracing/opentracing-go
+  version: ~0.9
```

Comment:
We aren't going to be able to 1.0.0 until opentracing-go does, nor can we loosen this constraint because that would potentially allow breaking changes in.

I created https://github.com/opentracing/opentracing-go/issues/112 to follow up with the OpenTracing guys.